scim2-sdk-common/src/main/java/com/unboundid/scim2/common/utils/MapperFactory.java

```diff
@@ -152,9 +221,20 @@ public MapperFactory setSerializationCustomFeatures(
    *     and deserializing SCIM JSON objects.
    */
   @NotNull
-  public static ObjectMapper createObjectMapper()
+  public ObjectMapper createObjectMapper()
   {
-    ObjectMapper mapper = new ObjectMapper(new ScimJsonFactory());
+    // Create a new object mapper with case-insensitive settings.
+    var objectMapperBuilder = JsonMapper.builder(new ScimJsonFactory());
+
+    // Do not care about case when de-serializing POJOs.
+    objectMapperBuilder.enable(ACCEPT_CASE_INSENSITIVE_PROPERTIES);
```

Comment:
The original mapper.configure() call for this property (which I've now deleted) was marked as deprecated by Jackson, so I've updated the logic to instead set the property on an object mapper builder instead. I'm not sure why other setting types (e.g., deserialization features) are okay with setting this directly on an object mapper, but it seems to be what the library encourages.